plugins/modules/katello_manifest.py

```diff
@@ -118,26 +115,29 @@ def main():
                 if 'repository_url' in entity_dict:
                     params['repository_url'] = entity_dict['repository_url']
                 params.update(scope)
-                changed, result = module.resource_action('subscriptions', 'upload', params, files=files)
+                # TODO: add a no_change field to resource_action
+                saved_changed = module.changed
+                result = module.resource_action('subscriptions', 'upload', params, files=files)
                 for error in result['humanized']['errors']:
                     if "same as existing data" in error:
-                        changed = False
+                        # We should not do this, but i see no better option
+                        # "Katello api is a sausage"
+                        module._changed = saved_changed
```

Comment:
This is the only ugly hack left. I think, it is either this, or the aforementioned `no_change` flag in `resource_action`.

Reply:
Added the `record_changed` flag.